network/dnsimple.py

```diff
@@ -294,7 +294,8 @@ def main():
             else:
                 module.fail_json(msg="'%s' is an unknown value for the state argument" % state)
 
-    except DNSimpleException, e:
+    except DNSimpleException:
+        e = get_exception()
```

Comment:
Yep, it work without (due to that code being already imported), but we should import something. I will add this.